src/core/annotation.js

```diff
@@ -634,7 +635,7 @@ var LinkAnnotation = (function LinkAnnotationClosure() {
         if (!isValidUrl(url, false)) {
           url = '';
         }
-        // According to ISO 32000-1:2008, section 12.6.4.7, 
+        // According to ISO 32000-1:2008, section 12.6.4.7,
```

Comment:
This change is no longer necessary when you rebase the patch as it has been fixed already in another commit.